src/Illuminate/Mail/Transport/Transport.php

```diff
@@ -67,4 +67,19 @@ protected function beforeSendPerformed(Swift_Mime_Message $message)
             }
         }
     }
+
+    /**
+     * Get the number of recipients.
+     *
+     * @param  \Swift_Mime_Message  $message
+     * @return int
+     */
+    protected function getCount(Swift_Mime_Message $message)
+    {
+        $contacts = array_merge(
+            (array) $message->getTo(), (array) $message->getCc(), (array) $message->getBcc()
+        );
+
+        return count($contacts);
```

Comment:
This is making an assumption that the send process always going to successfully send to every recipients. 

> When using send() the message will be sent just like it would be sent if you used your mail client. An integer is returned which includes the number of successful recipients. If none of the recipients could be sent to then zero will be returned, which equates to a boolean false. If you set two To: recipients and three Bcc: recipients in the message and all of the recipients are delivered to successfully then the value 5 will be returned.
>
> http://swiftmailer.org/docs/sending.html